open_humans/serializers.py

```diff
@@ -45,3 +74,25 @@ def get_sources(obj):
         sources = (k for k, i in get_source_labels_and_names_including_dynamic()
                    if project_membership_visible(obj.member, k))
         return sorted(sources)
+
+
+class DataUsersBySourceSerializer(serializers.ModelSerializer):
+    """
+    Serialize the members of a data source.
+    """
+
+    class Meta:  # noqa: D101
+        model = DataRequestProject
+        fields = ('id', 'name')
+
+    def to_representation(self, data):
+        ret = OrderedDict()
+        fields = self.get_fields()
+        query_params = dict(self.context.get('request').query_params)
+        source = 'direct-sharing-{}'.format(str(getattr(data, 'id')))
+        usernames = using_badges_to_track_project_membership_must_die(source)
```

Comment:
Because DataRequestProject doesn't contain a list of members.

I have a plan to deal with the current reliance on badges.  I want to keep it a separate project for now, though.  I'll use the information in DataRequestProjectMember to determine project membership.  It's literaly the point of the table.  I don't plan to keep the badges column in Member, as each member's list of badges can be constructed based on project membership.

Reply:
I have started working on the badgectomy.  I am doing the work in a separate branch, and will open a new pull request for that.

Reply:
okay, looking around, there's only a couple of places where badges are still used, so I guess I'll go ahead and do that thing.